apps/frontend/src/modules/groups/slices/group.slice.ts

```diff
@@ -32,6 +32,13 @@ const { actions, name, reducer } = createSlice({
 			state.groups = [];
 			state.dataStatus = DataStatus.REJECTED;
 		});
+		builder.addCase(deleteById.fulfilled, (state, action) => {
+			const deletedGroupId = action.meta.arg;
+			state.groups = state.groups.filter(
+				(group) => group.id !== deletedGroupId,
+			);
+			state.groupsTotalCount -= 1;
```

Comment:
```suggestion
			state.groupsTotalCount -= GROUPS_DELETE_COUNT;
```

Reply:
Do we really need to keep track `groupsTotalCount` in state? Can't we use `groups.length` instead? Or this may lead to unwanted rerenders?

Reply:
@Fjortis It has an effect on table pagination. `groups.length` will only show the number of elements on a page.